### PR TITLE
Blocks: Splitting post-list InspectorControls components

### PR DESCRIPTION
**Pull request by ryelle**
See #167 for background + merge plan

Split out the few components in `source/components/post-list/inspector-controls.js` into a new folder `source/components/post-list/inspector-controls/` with one file per component. Also adds PropType definitions to each component.

### CODE REVIEW

**Comment by ryelle** [comment]
After discussion in #167, this should get updated to merge the two components into one instead of separating into different files.

File: public_html/wp-content/mu-plugins/blocks/source/components/post-list/inspector-controls.js [hunk 1]
```diff
@@ -1,75 +1,61 @@
...
+ */
+import { get }   from 'lodash';
+import PropTypes from 'prop-types';
+
 /**
  * WordPress dependencies
  */
-import { PanelBody, RangeControl } from '@wordpress/components';
-import { Component, Fragment }     from '@wordpress/element';
+import { RangeControl, PanelBody } from '@wordpress/components';
```
Comment (coreymckrill) [inline public_html/wp-content/mu-plugins/blocks/source/components/post-list/inspector-controls.js]: Just curious why the order of listed components is changed here.

**Review by coreymckrill — APPROVED** [review]

File: public_html/wp-content/mu-plugins/blocks/source/components/post-list/inspector-controls.js (hunk 1 above)
Comment (ryelle) [reply public_html/wp-content/mu-plugins/blocks/source/components/post-list/inspector-controls.js]: That's a great question - either a merge conflict or I forgot how the alphabet works 😆

Comment (coreymckrill) [reply public_html/wp-content/mu-plugins/blocks/source/components/post-list/inspector-controls.js]: 😆

**Pull request merged**